Aufg6/WieseIdee.ts

```diff
@@ -29,11 +29,12 @@ namespace Aufg6_Wiese {
         inhalt = canvas.getContext("2d");
         
         for (let i: number = 0; i < n; i++) {
-            b.x = Math.random() * 200;
+            b.x = Math.random() * 200; <-- ahh, hier wird b genutzt. Aber so wird das nicht funktionieren...
             b.y = Math.random() * 200; 
             b.g = Math.random() * 30 + 10;
             b.farbe = "hsl(" + Math.random() * 360 + ", 100%, 50%)";
-            bees[i] = b;

```

Comment:
"   for (let i: number = 0; i < n; i++) {
            let s: Square = { x: 0, y: 0, size: 0, color: "#0000ff" }; // default-values
            s["x"] = Math.random() * 200;
            s.y = Math.random() * 200;
            s.size = Math.random() * 30 + 10;
            s.color = "hsl(" + Math.random() * 360 + ", 100%, 50%)";
            squares[i] = s;"

habe ich versucht ^^'